src/api/common/schemas/get-ledger.json

```diff
@@ -14,7 +14,12 @@
     "parentLedgerHash": {"$ref": "hash256"},
     "parentCloseTime": {"type": "integer", "minimum": 0},
     "totalDrops": {"$ref": "value"},
-    "transactionHash": {"$ref": "hash256"}
+    "transactionHash": {"$ref": "hash256"},
+    "transactions": {"type": "array", "items": {"type": "object"}},
+    "rawTransactions": {"type": "string"},
+    "transactionHashes": {"type": "array", "items": {"$ref": "hash256"}},
+    "rawAccounts": {"type": "string"},
```

Comment:
Out of scope of this P.R. but have you considered making it so that it's possible to convert losslessly between rippled <-> RippleAPI json formats?